assignments/mgmt-groups/mg-dts002/assign.appgateway_diagnostics.json

```diff
@@ -13,21 +13,18 @@
                 "value": "/subscriptions/8ae5b3b6-0b12-4888-b894-4cec33c92292/resourceGroups/soc-core-infra-prod-rg/providers/Microsoft.EventHub/namespaces/soc-prod-eventhubns/authorizationRules/RootManageSharedAccessKey"
             }
         },
-        "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/dts002/providers/Microsoft.Authorization/policyDefinitions/HMCTSAppGatewayDiagnostic",
+        "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/dts002/providers/Microsoft.Authorization/policyDefinitions/HMCTSAGWDiagnostic",
         "scope": "/providers/Microsoft.Management/managementGroups/dts002"
     },
     "location": "uksouth",
     "identity": {
-        "type": "UserAssigned",
-        "userAssignedIdentities": {
-            "/subscriptions/8ae5b3b6-0b12-4888-b894-4cec33c92292/resourceGroups/soc-core-infra-prod-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/soc-prod-eventhub-azure-policy": {}
-        }
+        "type": "SystemAssigned"
```

Comment:
why switch?

Reply:
need the Azure Event Hubs Data Owner” role, raised a pr to add the role to the uami but there’s a script to check there’s no owner roles ..and we want least privilege for that so seemed better to just use system assigned here

Reply:
can probably just remove that check it's very outdated and was more for the exact role 'Owner' not other ones

Reply:
the uami was originally made bc the system assigned only grants permissions to the target subscription which doesn't work if the event hub is in another sub. that issue doesn't apply here so are we wanting to use a uami just for consistency?